### PR TITLE
Add rubygem-vault package

### PR DESCRIPTION
**Pull request by timogoebel**
Please build for:

* [x] Nightly
* [x] 1.21
* [x] 1.20

This is a dependency for #3503

### CODE REVIEW

**Comment by timogoebel** [comment]
I used `./add_gem_package.sh vault scl foreman-plugins-nightly-rhel7`.

Added it manually to `package_manifest.yaml`.

**Comment by ekohl** [comment]
Yes, I still need to automate that part. I just merged https://github.com/theforeman/foreman-packaging/pull/3342 which at least adds the base and implements it for NPM. I'll take a stab at doing the same for gems.

**Comment by timogoebel** [comment]
@ekohl: Thanks, the tool should also check if the ini-tooling is installed. And that set is actually gnu sed. ;-)
I'll send patches if I find the time.

**Comment by timogoebel** [comment]
Repoclosure fails. I need some guidance here 😊

File: packages/plugins/rubygem-vault/rubygem-vault.spec [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Group: Development/Languages
+License: MPL-2.0
+URL: https://github.com/hashicorp/vault-ruby
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}ruby
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix}rubygem(aws-sigv4)
```
Comment (ekohl) [inline packages/plugins/rubygem-vault/rubygem-vault.spec]: This package is also needed and is causing the repoclosure failure.

**Comment by timogoebel** [comment]
Rebased.

**Comment by dgoetz** [comment]
[test rpm]

**Comment by dgoetz** [comment]
Looks good to me.
Test runs fine except from very last step "Notifies GitHub of the status of a Pull Request" which fails.

**Comment by timogoebel** [comment]
[test rpm]

**Comment by timogoebel** [comment]
[test rpm]

**Comment by timogoebel** [comment]
[test rpm]

**Comment by evgeni** [comment]
```
% koji add-pkg foreman-plugins-nightly-rhel7 tfm-rubygem-vault --owner=kojiadm                                                                                                                                                      
Adding 1 packages to tag foreman-plugins-nightly-rhel7
```

**Pull request merged**